csv2docx/cli.py

```diff
@@ -14,6 +14,14 @@
 @click.option(
     '--delimiter', '-d',
     default=";",
-    help='delimiter used in your csv. Default is \';\'')
-def main(data, template, delimiter):
-	c2d.convert(data, template, delimiter)
+    help='Delimiter used in your csv. Default is \';\'')
+@click.option(
+    '--name', '-n',
+    required=True,
+    help='Naming scheme for output files.')
```

Comment:
Perhaps we can expand on the help by adding:
`Specific column name to be used in the naming scheme for output files. '`

Reply:
Also - is this case sensitive?

Reply:
At the moment, yes. Let's discuss in #25